releasenotes/notes/ambient-hostnetwork.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: release-notes/v2
+kind: feature
+area: traffic-management
+
+releaseNotes:
+  - |
+    **Fixed** an issue causing `hostNetwork` pods to be ignored in ambient mode.
```

Comment:
```suggestion
    **Fixed** an issue causing `hostNetwork` upstream pods to be ignored in ambient mode.
```

Reply:
I wanna be a bit careful here - we explicitly do _not_ support adding hostNetwork pods to the ambient mesh and I don't want to confuse anyone into thinking we do.